docs/src/cluster/turbine-block-propagation.md

```diff
@@ -2,27 +2,27 @@
 title: Turbine Block Propagation
 ---
 
-A Solana cluster uses a multi-layer block propagation mechanism called _Turbine_ to broadcast transaction shreds to all nodes with minimal amount of duplicate messages. The cluster divides itself into small collections of nodes, called _neighborhoods_. Each node is responsible for sharing any data it receives with the other nodes in its neighborhood, as well as propagating the data on to a small set of nodes in other neighborhoods. This way each node only has to communicate with a small number of nodes.
+A Solana cluster uses a multi-layer block propagation mechanism called _Turbine_ to broadcast transaction shreds to all nodes with minimal amount of duplicate messages. The cluster divides itself into small collections of nodes, called _neighborhoods_. Each node is responsible for propagating any data it receives on to a small set of nodes in downstream neighborhoods and possibly sharing data with the other nodes in its neighborhood. This way each node only has to communicate with a small number of nodes.
```

Comment:
> and possibly sharing data with the other nodes in its neighborhood

I think this is both confusing and soon outdated.
So again, maybe just describe the new logic.

Reply:
Tried to add some language to clarify the special role of anchor nodes.

I'm thinking to describe the current behavior of MNB and I'm happy to have a fast follow update once redundant turbine path is removed

Reply:
If that works, that's simple enough. I'm a little skeptical because I don't see the existing turbine images in that directory:
```
/img/data-plane-seeding.svg
/img/data-plane-fanout.svg
/img/data-plane.svg
/img/data-plane-neighborhood.svg
```